tests/purs/docs/src/DocCommentsDataConstructor.purs

```diff
@@ -0,0 +1,5 @@
+module DocCommentsDataConstructor where
+
+data Foo
```

Comment:
Could you please add some more tests? I think we ought to have at least:

- One where the data type has some type variables,
- One where there are more data constructors, where some have comments and some don't,
- One where the data constructors have one or more fields.
- One which is a `newtype`